ChatQnA/langchain/docker/docker-compose.yml

```diff
@@ -23,7 +23,9 @@ services:
     build:
       args:
         https_proxy: ${https_proxy}
+        http_proxy: ${http_proxy}
       dockerfile: Dockerfile
+      context: .
```

Comment:
Is this force required?

Reply:
this is just telling docker compose where to look for Dockerfile to build the image. it was erroring out without it